test/functional/environments_controller_test.rb

```diff
@@ -1,16 +1,12 @@
 require 'test_helper'
+require 'functional/shared/basic_rest_response_test'
 
 class EnvironmentsControllerTest < ActionController::TestCase
-  test "should get index" do
-    get :index, {}, set_session_user
-    assert_response :success
-    assert_not_nil assigns(:environments)
```

Comment:
It's not great that these assertions are removed, they can help check that the index is generating the right content.  (Same for one or two others in single-resource tests.)